### PR TITLE
conv2d

### PR DESCRIPTION
**Pull request by rum1887**
Add Layers to Paddle Backend #10696

### CODE REVIEW

**Comment by rum1887** [comment]
hi, @MahmoudAshraf97 kindly review this pr :)

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]
paddle already has data format argument, why not use it directly instead of the axis swap?

**Comment by rum1887** [comment]
> paddle already has data format argument, why not use it directly instead of the axis swap?

yeah, so I initialled the data format to "NCHW" (PyTorch, Paddle has this format as the default whilst Tensorflow has "NHCW" as the default dataformat). 

I changed the code to perform axis swap once at the end.

works?

**Comment by rum1887** [comment]
> > paddle already has data format argument, why not use it directly instead of the axis swap?
> 
> Yes!
> 
> I initialized the data format to "NCHW". PyTorch, Paddle has this format as the default whilst Tensorflow has "NHCW" as the default dataformat.
> 
> I changed the code to perform axis swap once at the end.
> 
> works?

umm, since Paddle produces output consistent with the input format passed, I removed the axis swaps.

File: ivy/functional/backends/paddle/layers.py [hunk 1]
```diff
@@ -118,8 +118,22 @@ def conv2d(
     data_format: str = "NHWC",
     dilations: Union[int, Tuple[int, int]] = 1,
     out: Optional[paddle.Tensor] = None,
-) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+):
+    df = "NHWC"
+    if data_format == "NCHW":
+        df = "NCHW"
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/layers.py:124]: I still don't see a need for this, what does this snipped add?

**Comment by rum1887** [comment]
> please use the hypothesis tests to make sure your implementation is correct before requesting another review

yes, will do that!

why are the axes of filters switched in conv3d?

File: ivy/functional/backends/paddle/layers.py (hunk 1 above)
Comment (rum1887) [reply ivy/functional/backends/paddle/layers.py]: yeah, its nit needed
